drivers/backup/backup.go

```diff
@@ -194,6 +194,16 @@ type ScheduleBackup interface {
 
 	// DeleteBackupSchedule
 	DeleteBackupSchedule(req *api.BackupScheduleDeleteRequest) (*api.BackupScheduleDeleteResponse, error)
+
+	// BackupScheduleWaitForNBackupsCompletion, waits for backup schedule to complete successfully
+	// or till timeout is reached. API should poll every `timeBeforeRetry` duration
```

Comment:
Could you put context.Context as a first argument to these functions. It can help with tracking timeouts and cancellation. how do you think?

Reply:
Fixed.